src/vivarium_csu_ltbi/tools/results.py

```diff
@@ -78,6 +81,9 @@ def get_common_seeds(df: pd.DataFrame) -> set:
     seed_sets = []
     for draw, g in df.groupby(['input_draw']):
         if 'scenario' in g.columns:  # We have an additional criterion
```

Comment:
I don't think this is right. Not adding a set of seeds for this draw is equivalent to adding a set of all seeds for the draw. 
Simple example:
```
set.intersection(*[{1, 2, 3}, {2, 3, 4}, {1, 2, 3, 4}]) == {2, 3}
set.intersection(*[{1, 2, 3}, {2, 3, 4}]) == {2, 3}
```
That second line is what is happening here if I'm not mistaken